config/database.yml

```diff
@@ -22,9 +22,8 @@ test: &test
   database: <%= ENV['POSTGRES_TEST_DB'] || 'theodinproject_test' %>
 
 docker:
-  <<: *default
+  <<: *development
```

Comment:
If we change this to `*deveopment` I don't think we need the following `database: ... ` line as it's  a duplicate of Development

Reply:
It didn't work without it